supporting_content/columns/chargecategory.md

```diff
@@ -0,0 +1,98 @@
+# Column: ChargeCategory

```

Comment:
It looks like when we renamed ChargeType to ChargeCategory we missed the supporting content. Can you merge this into chargetype.md and then rename that to chargecategory.md?

Reply:
done in this pr